### PR TITLE
Add time event

### PR DESCRIPTION
**Pull request by mixpanel-andrew**
adds time_event method

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+**2.6.0** (5 Aug 2015)
+- add `time_event` method
+- update snippet to load mixpanel correctly from cdn when ﬁle is opened locally
```
Comment (tdumitrescu) [inline CHANGELOG.md]: super nitpick but let's not ligate the fi of 'file' please!

Comment (Quuxplusone) [inline CHANGELOG.md]: digression @zakj we could have a git hook that requires text files to contain only ASCII.
We probably have some unit-tests today that use raw Unicode, but they should all be changed to use `\u` and so on, anyway (so that we only have to trust "does Python handle \u sequences correctly", as opposed to "does every text editor and source code tool in the world handle non-ASCII bytes correctly").

Comment (zakj) [inline CHANGELOG.md]: I wouldn't want to forbid utf8 in HTML files. I would argue that utf8 in `.py` files is not the end of the world either, so long as there's a corresponding `coding: utf-8` comment. I'd be down to warn on utf-8 in `.txt`/`.md` though ;)

Comment (Quuxplusone) [inline CHANGELOG.md]: Oh, it's never the end of the world. It tends to only matter in insane corner cases, like "oops, my editor replaced U+2009 with U+0020 and broke that test" or "oops, Phabricator threw out half of my diff because MySQL truncated the whole thing at the first U+1F600" — i.e., it's things that all reasonable people would agree don't belong in source code. Very occasionally it matters when "oops, I opened my unit test in Word and suddenly `assertEqual(output, "ﬁle not found")` is failing", in which case nothing *super* insane is going on (except someone using Word, of course) but the git hook would still catch the mistake.

I'd definitely agree that .htm/.html files get a pass.

Comment (zakj) [inline CHANGELOG.md]: Fair points. Should just be a matter of writing the appropriate function and adding it to `VALIDATIONS` in the main block of `pre-commit`; I won't have time to work on it anytime soon but would be happy to thumbs-up your PR!

Comment (Quuxplusone) [inline CHANGELOG.md]: :bluffscalled: :)
